go-controller/go.mod

```diff
@@ -34,3 +36,5 @@ require (
 	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
 	sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e // indirect
 )
+
+replace github.com/ebay/go-ovn v0.1.0 => github.com/ebay/go-ovn v0.0.0-20200604015400-fff9cdb
```

Comment:
what does this do? I was trying to run a test with your PR pulled into my local system and I see the below error

```
[openshift@nfvha-comp-01 go-controller]$ go test -v -run TestNextIP
go: github.com/ebay/go-ovn@v0.0.0-20200604015400-fff9cdb: invalid pseudo-version: revision is shorter than canonical (fff9cdbba909)
```
What steps should I take to resolve the above when running unit tests?

Reply:
I think maybe a recent go version checks for revision to be canonical. let me see if I can fix that.